### PR TITLE
Readme final, Rubocop fixes

### LINKED ISSUE #98: Final touches on README
- [ ] Update schema picture

### PR DESCRIPTION
**Pull request by LHJE**
### Code Highlights:
 * Updates the `README.md`
 * Updates `dates_in_past_validator.rb` to appease rubocop errors
### Where should the reviewer start?
 * `README.md`
 * `dates_in_past_validator.rb`
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.
### Issues:
* Addresses #98
* Closes #98
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations: Just 2 `get_` errors

### CODE REVIEW

File: app/models/concerns/dates_in_past_validator.rb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
   end
+
+  def era(record)
+    record.errors.add(:start_date, 'cannot be in the future') if record.start_date > Time.zone.today
+    record.errors.add(:end_date, 'cannot be in the future') if record.end_date > Time.zone.today
+  end
+
+  def event(record)
+    record.errors.add(:date, 'cannot be in the future') if record.date > Time.zone.today
+  end
```
Comment (LHJE) [inline app/models/concerns/dates_in_past_validator.rb:19]: This was so rubocop stopped giving us 7 errors.  Now it's just the 2 `get_` errors

**Review by LHJE — COMMENTED** [review]
We should go over any other changes we might want!

File: README.md [hunk 2]
```diff
@@ -244,12 +263,16 @@ With Eras, you'll see your life in a whole new way.
 
 ## Database Schema
 - ![our schema](/schema.png)
-- Insert description of each table
+- Insert description of each table
```
Comment (LHJE) [inline README.md]: We're gonna need to update this part too

File: README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ With Eras, you'll see your life in a whole new way.
 - [Contributors](#contributors)
 
 ## User Interface
-- [Deployed application](insert link)
```
Comment (leahriffell) [inline README.md:22]: We can update this with the FE heroku app once they've deployed

File: app/models/concerns/dates_in_past_validator.rb [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
     if record.instance_of?(Era) && record.start_date && record.end_date
-      record.errors.add(:start_date, 'cannot be in the future') if record.start_date > Date.today
-      record.errors.add(:end_date, 'cannot be in the future') if record.end_date > Date.today
+      era(record)
     elsif record.instance_of?(Event) && record.date
-      record.errors.add(:date, 'cannot be in the future') if record.date > Date.today
+      event(record)
     end
   end
+
```
Comment (dhalverson) [inline app/models/concerns/dates_in_past_validator.rb:9]: Should these helpers be private?

Comment (dhalverson) [reply app/models/concerns/dates_in_past_validator.rb:9]: Should these be added to a permit type method, to only allow dates and no other types of data to be passed into our method?

More of a security feature thought.

**Review by dhalverson — APPROVED** [review]
Approved, left a few comments that don't necessarily need to be resolved

**Pull request merged**